add kubeconfig option to avoid prompt (#58)
codeready-toolchain/toolchain-common: README.adoc

    . creates a secret in `toolchain-member-operator` namespace with the SA token taken from the `toolchain-host-operator` ns
    . creates `KubeFedCluster` in `toolchain-member-operator` namespace CR representing the added `host`

==== Using a different clusters for host and member operators

> Make sure to have two different clusters ready and available. To setup Toolchain, you'll have to run `host-operator` and `registration-service` on host cluster and `member-operator` on member cluster.

Reviewer: I don't think that we need to mention the registration service here as there is no relation to the script. In addition, the reg-service will be part of the host operator
```suggestion
Make sure you have two different clusters ready and available where the `host-operator` should be running in the host cluster and `member-operator` in the member cluster.
```
